pkg/scheduler/core/generic_scheduler.go

```diff
@@ -429,7 +429,7 @@ func (g *genericScheduler) prioritizeNodes(
 
 	if klog.V(10).Enabled() {
 		for i := range result {
-			klog.Infof("Host %s => Score %d", result[i].Name, result[i].Score)
+			klog.InfoS("Calculated final score for node", "node", result[i].Name, "score", result[i].Score)
```

Comment:
Please add the pod. Sorry I missed this before.

Reply:
Done.